src/app/wallet/wallet/receive/receive.component.html

```diff
@@ -132,16 +132,6 @@
     <!-- use two paginator for reset pagination after tab change. -->
     <div class="receive-pagination">
       <app-paginator
-        #paginator
-        *ngIf="type == 'private' && getSinglePage().length > 0"
-        [length]="getTotalCountForPagination()"
-        [pageSize]="MAX_ADDRESSES_PER_PAGE"
-        [pageSizeOptions]="PAGE_SIZE_OPTIONS"
-        (page)="pageChanged($event)">
-      </app-paginator>
-
-      <app-paginator
-        *ngIf="type == 'public' && getSinglePage().length > 0"
         #paginator
         [length]="getTotalCountForPagination()"
```

Comment:
I believe `getSinglePage().length > 0` this should be their as it only show the pagination when data is available

Reply:
changes has been completed

Reply:
Ah, yes, thanks for spotting that @vikas-cis  :)

@EagleEye25 please include the condition for app-paginator for
`*ngIf="getSinglePage().length > 0"`